__init__.py

```diff
@@ -376,6 +376,7 @@ def handle_set_alarm(self, message):
                 )
 
         self._show_alarm_anim(alarm_time)
+        self._show_alarm_ui(alarm_time, name)
```

Comment:
wondering if we bump this above the `_show_alarm_anim()` as that method is blocking whilst `_show_alarm_ui()` shoudn't be.